src/test/rules/terComputedPropertySpacingRuleTests.ts

```diff
@@ -0,0 +1,167 @@
+import { RuleTester, Failure, Position, dedent } from './ruleTester';
+
+const ruleTester = new RuleTester('ter-computed-property-spacing');
```

Comment:
If you do decide to add the fixer you'll have to add the second parameter to the rule tester. At the moment not all the rules have a fixer and I have included the expected output to some tests. Here we explicitly tell the tester that we want to test the fixer. For instance:

```ts
const ruleTester = new RuleTester('ter-arrow-spacing', true);
```